frontend/src/locales/en/translation.json

```diff
@@ -725,7 +725,7 @@
     "Published": "Published",
     "Archived": "Archived",
     "Copy": "Copy",
-    "ThisWillCopy": "This will copy",
+    "ThisWillCopy": "This will create a draft copy of",
```

Comment:
This change is based on Jason's request. He wants the statement "This will create a draft copy of" to be used when asking for confirmation to copy a dashboard whatever the status of the original dashboard is. I do not see a need to have two entries for the same statement. If this changes in the future, we can add a new statement entry then.